Makefile

```diff
@@ -2,210 +2,220 @@ ifneq (,)
 .error This Makefile requires GNU Make.
 endif
 
-# -------------------------------------------------------------------------------------------------
-# Default configuration
-# -------------------------------------------------------------------------------------------------
-.PHONY: lint build rebuild test tag pull-base-image login push enter
+# Ensure additional Makefiles are present
```

Comment:
Maybe include link to repo: https://github.com/devilbox/makefiles/
I was looking for the docker build part and didn't immediately see it.